src/vm/checker/typecheck/tests/contracts.rs

```diff
@@ -7,21 +7,21 @@ use vm::checker::{AnalysisDatabase, AnalysisDatabaseConnection};
 
 const SIMPLE_TOKENS: &str =
         "(define-map tokens ((account principal)) ((balance int)))
-         (define-read-only (get-balance (account principal))
+         (define-read-only (my-get-token-balance (account principal))
```

Comment:
We could revert these changes.

Reply:
True, but I left it as is, because this is just test code. However, I did revert the sample programs.